### ISSUE TITLE
Calendar and contacts

### ISSUE THREAD

**Issue by mkwork** [VolkMilit/CuteMail#20]
Are you planning on this?

**Comment by VolkMilit** [comment]
No, I'm not planing calendar. Well, at least, maybe after 1.0. See goals in readme, I just want lightweight email client.
About contacts: yes.
